qt/DetailsDialog.cc

```diff
@@ -1237,6 +1310,8 @@ void DetailsDialog::refreshUI()
     }
 
     setEnabled(true);
+
+    labels_need_refresh_ = false;
```

Comment:
_(nit)_ move this line to the beginning of the `if (labels_need_refresh_)` clause at line 895 just to keep related pieces grouped together